audbackend/core/backend.py

```diff
@@ -286,73 +308,6 @@ def ls(
         """
         return sorted(self._ls(path))
```

Comment:
For `ls()` we are missing a check if the path contains any non allowed chars. I guess you do not check here as it otherwise does not need `utils.check_path_and_ext()`

Reply:
I know that you didn't touched it. I just ask for the check to be consistent as with `split()`. 
I created https://github.com/audeering/audbackend/issues/48 so we don't need to change it here.